assets/src/js/header.js

```diff
@@ -54,10 +50,19 @@ export const setupHeader = function($) {
 
   // Close all menus when clicking somewhere else
   $(document).on('click', function closeInactiveMenus(evt) {
+    let searchToggled = false;
     const clickedElement = evt.target;
     $('button[aria-expanded="true"]').each((i, button) => {
       const $button = $(button);
       const buttonTarget = $($button.data('bs-target')).get( 0 );
+
+      if ($button.get(0).classList.contains('nav-search-toggle')) {
```

Comment:
Shouldn't we instead tackle the root cause, which is the usage of `.trigger('click')`?

Reply:
My view was more to do that when we rewrite this script without jquery

Reply:
As discussed on Slack, the script needs a significant rewrite, so we can go with this solution for now.

Another way would be to only select visible buttons.
```js
$('button[aria-expanded="true"]:visible')